### PR TITLE
App router tests

### PR DESCRIPTION
**Pull request by igorosip0v**
- Adds tests from `/web`
- Removes CI file for `/web`
  - GitHub Actions is now working only for the /web-app-router
  - I will adjust it once more after move all app router files back to `/web`
- Fixes tests
  - Tests currently have imports from the`/legacy` folder. We will need to adjust test once more when we will refactor /pages/api

### CODE REVIEW

File: web-app-router/next.config.mjs
Comment (andy-t-wang) [inline web-app-router/next.config.mjs:1]: remove thi s

**Review by andy-t-wang — APPROVED** [review]
Added small comments looks fine if you just copied tests over

File: web-app-router/pages/api/v1/precheck/[app_id].ts [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
-
 import {
   ActionModel,
   AppMetadataModel,
   AppModel,
   NullifierModel,
 } from "@/legacy/lib/models";
 import { NextApiRequest, NextApiResponse } from "next";
-import { CanUserVerifyType, EngineType } from "@/lib/types";
+import { CanUserVerifyType, EngineType } from "@/legacy/lib/types";
```
[the remark this answers is not in the document]
Comment (igorosip0v) [reply web-app-router/pages/api/v1/precheck/[app_id].ts:14]: It's still a legacy page. And also it caused an error in tests

File: web-app-router/next.config.mjs
Comment (igorosip0v) [reply web-app-router/next.config.mjs:1]: We need this for jest.setup.ts

Comment (andy-t-wang) [reply web-app-router/next.config.mjs:1]: oh ok

**Pull request merged**